README.md

```diff
@@ -132,33 +132,35 @@ The trace configuration additionally exposes the `samplingRate` option which set
 
 ## Custom Tracing API
 
-The custom tracing API can be used to add custom spans to trace. A *span* is a particular unit of work within a trace, such as an RPC request. Currently, you can only use the custom tracing API inside the following web frameworks: `express`, `hapi`, `restify`.
+The custom tracing API can be used to add custom spans to trace. A *span* is a particular unit of work within a trace, such as an RPC request. Spans may be nested; the outermost span is called a *root span*, even if there are no nested child spans. In general, root spans typically correspond to incoming requests, while *child spans* typically correspond to outgoing requests, or other work that is triggered in response to incoming requests.
```

Comment:
I think it is necessary to note this since I mention an 'outermost' span immediately afterwards. It doesn't say anywhere else in the document that spans may be nested, and it may not be immediately obvious from its standard usage in web frameworks, since users never have to deal with root spans on their own.